.github/workflows/sync_project.yml

```diff
@@ -0,0 +1,34 @@
+name: cookietemple sync
```

Comment:
why is this necessary? did we missed the file in initial release?

Reply:
cookietemple itself did not have the sync workflow :)

Reply:
Ah makes sense ty